src/features/commands/buttons/doNothing.js

```diff
@@ -0,0 +1,17 @@
+const { Client } = require('discord.js'),
+    db = require('../../../db'),
+    perspective = require('../../../api/perspective');
```

Comment:
Unnecessary dependencies
```suggestion
const { Client } = require('discord.js');
```